### ISSUE TITLE
 Support stylelint 11

### ISSUE THREAD

**Issue by limonte** [csstree/stylelint-validator#20]
After upgrading stylelint to 11, I'm getting this warning:

```
warning "stylelint-csstree-validator@1.5.2" has incorrect peer dependency "stylelint@>=7.0.0 <11.0.0".
```

**Comment by limonte** [comment]
Done in https://github.com/csstree/stylelint-validator/commit/9ad722f11e056fdf6790b776f947d3494ba0f821

**Comment by lahmatiy** [comment]
Please, use `stylelint-csstree-validator@1.6.1` with stylelint 11
